### ISSUE TITLE
how to set a constant learning rate ?

### ISSUE THREAD

**Issue by fitoule** [google/automl#572]
how to set a constant learning rate ?

**Comment by mingxingtan** [comment]
A simple way workaround is set:

h.lr_warmup_init = h.learning_rate
h.lr_warmup_epoch = h.num_epochs

**Comment by fitoule** [comment]
Actually it does'nt work for me. My batchSize is 24
[url=http://www.noelshack.com/2020-28-3-1594199551-screenshot-2020-07-08-tensorboard.png][img]http://image.noelshack.com/minis/2020/28/3/1594199551-screenshot-2020-07-08-tensorboard.png[/img][/url]
my command line
python3 main.py --mode=train_and_eval --training_file_pattern=/home/test/train/OID*.record --validation_file_pattern=/home/test/val/* --model_name=efficientdet-d1 --model_dir=model/person/D1/LR0015E5_fr1_nE19000_BS24 --ckpt=ref_backbone/efficientdet-d1 --train_batch_size=24 --num_examples_per_epoch=19000 --num_epochs=5 --eval_batch_size=8 --eval_samples=80 --hparams="num_classes=1,learning_rate=0.01,lr_warmup_init=0.01,lr_warmup_epoch=5,max_instances_per_image=200,var_freeze_expr='(efficientnet)'";

**Comment by mingxingtan** [comment]
Oh, you are right,  lr_warmup_init is adjusted based on batch size.

I just added constant LR, you can specify h.lr_decay_method='constant'
